docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       dockerfile: .docker/Dockerfile
-    command: rails s -p 3000 -b 0.0.0.0
+    command: sh -c "yarn install && rails s -p 3000 -b 0.0.0.0"
```

Comment:
I'd really like to keep it so the up command is just `rails s -p 3000 -b 0.0.0.0` to be consistent with more general rails behavior, and `yarn install` feels closer to a `bundle install` than it does booting up puma. How much of a pain would it be to move this command to the dockerfile?

Reply:
To do that we either need an `entrypoint.sh`, or something akin. The reason we have to do this is because the `yarn.lock` is copied into the Dockerfile with `COPY .` and this ensures that the yarn.lock integrity is valid. We could in theory exclude the yarn.lock from being copied in but I don't know how much I like that either. Open to ideas!

Reply:
I did some cursory googling and this seems not totally wild: https://medium.com/soulmates-ai/running-a-rails-app-with-webpacker-and-docker-8d29153d3446 Specifically the part about:

```
# config/environments/development.rb
config.webpacker.check_yarn_integrity = false
```